src/diag.rs

```diff
@@ -1165,6 +1166,12 @@ impl Diagnostic {
                 stmt,
                 *tok,
             )]),
+            UnknownCommandToken(span) => ("Unknown command token".into(), vec![(
+                AnnotationType::Warning,
+                "This token was not declared in any $v or $c statement".into(),
+                stmt,
+                *span,
+            )]),
```

Comment:
Why is this not just `UnknownToken`? It should be an error condition (even though you won't get it unless you try to parse the grammar).

Reply:
`UnknownToken` already exists with a `TokenIndex` as parameter, while this version of the diagnostic has a `Span`. 

`UnknownCommandToken` is currently only used within the commands, because we have no token index there, but just a span. I'm not sure if it might be used elsewhere.

Do you have a better naming (which does not collide with `UnknownToken`)?

Reply:
I feel like something similar would have come up in typesetting comment parsing...

Reply:
Sounds good, I think I can refactor to use `UndefinedToken` instead.